data/Dels/Dels missions.txt

```diff
@@ -698,6 +698,163 @@ mission "First Contact: Dels Ufulre"
+#			`	You prepare to launch.`
+#				accept
+#	on abort
+#		"reputation: Dels Kaote" -= 1
+#		fine 10000
+#		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+#
+#	npc evade
+#		government "Dels Ufulre"
+#		personality
+#			confusion 30
+#			plunders harvests target # no staying so they can flee
+#		fleet "Dels Ufulre Raid" 2
+#		system destination
+#
+#	npc
+#		government "Dels Kaote"
+#		personaility
+#			confusion 10
+#			heroic opportunistic frugal staying
+#		fleet "Dels Kaote"
+#		system destination
+#	on complete
+#		payment 100000
+#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`
+
+mission "Dels Officer: Patrol Systems"
+	landing
+	repeat
+	priority
+	name `Partrol systems`
+	source Delro
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	stopover
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	description `You must patrol <stopovers> to check for Ufulre activity.`
+	to offer
+		has "dels officer: report mission available"
+	on offer
+		clear "dels officer: report mission available"
+		conversation
+			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <last>, you are requested for patrol on <stopovers> to check on Ufulre activity."`
+			`	You prepare to launch.`
+				accept
+	on abort
+		"reputation: Dels Kaote" -= 1
+		fine 10000
+		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+	on complete
+		payment 50000
+		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`
+
+mission "Dels Officer: Escort Convoy"
+	landing
+	repeat
+	priority
+	name `Escort convoy`
+	source Delro
+	destination
+		government "Dels Kaote" Dels
+		near Alkurud 0 2
+	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
+	to offer
+		has "dels officer: report mission available"
+	on offer
+		clear "dels officer: report mission available"
+		conversation
+			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <last>, you must escort a convoy to <destination> in order to assist those living there."`
+			`	You prepare to launch.`
+				accept
+	on abort
+		"reputation: Dels Kaote" -= 1
+		fine 10000
+		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
+
+	npc accompany save
+		government "Dels Kaote"
+		personaility
+			confusion 10
+			heroic opportunistic frugal staying

```

Comment:
They won't follow you if they have staying. Also remove heroic so they aren't suicidal.